spec/Machine/init.js

```diff
@@ -26,7 +26,28 @@ module.exports = (sampleFile) => {
     },
 
     getResultWithLineControl: () => {
-      return fs.readFileSync(sampleFile + '.out-lc', 'utf-8');
+      let content;
+      try {
```

Comment:
No need to invent a bicycle: https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js